src/entity/base_entity.rs

```diff
@@ -413,10 +498,69 @@ pub trait EntityTrait: EntityName {
     /// assert_eq!(
     ///     db.into_transaction_log(),
     ///     vec![Transaction::from_sql_and_values(
-    ///         DbBackend::Postgres, r#"UPDATE "fruit" SET "name" = $1 WHERE "fruit"."id" = $2 AND "fruit"."name" LIKE $3"#,
+    ///         DbBackend::Postgres,
+    ///         r#"UPDATE "fruit" SET "name" = $1 WHERE "fruit"."id" = $2 AND "fruit"."name" LIKE $3 RETURNING "id", "name", "cake_id""#,
```

Comment:
As in UPDATE?

Reply:
Is related to Postgres insert?

Reply:
Here the returning clause is returning all columns, which is different from the insert case where it only returns 'I'd'

Reply:
This is `update`'s docs.

> Here the returning clause is returning all columns, which is different from the insert case where it only returns 'id'

Yes! Any issue here?
Why don't we write it as `RETURNING *`?

Reply:
No issue here. I'd just want to see a consistent behaviour with insert. I could not find the counterpart test case.